src/components/network-modification-tree.js

```diff
@@ -77,8 +77,10 @@ const NetworkModificationTree = ({
             ) {
                 case 'BUILT':
                     return '#70d136';
-                case 'NOT_BUILT':
-                    return '#9196a1';
+                case 'BUILT_WITH_WARNING':
```

Comment:
For the colors, you can use the constants defined in network-modification-node.js

Reply:
Actually the colors was not the same that's why I didn't use constants, but IMO we should use the same everywhere

Reply:
I'd like this aspect to sneak in a theme for third mode after daylight and night modes